### PR TITLE
Update and rename ID to TOKEN.py to ID to TOKEN WITH THREADING.py

### PR DESCRIPTION
**Pull request by TheRealReed**
added threading myself 😈

### CODE REVIEW

File: ID to TOKEN WITH THREADING.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import random
 import string
 import requests
+import threading
 from colorama import Fore, init, Style
```
Comment (TheRealReed) [inline ID to TOKEN WITH THREADING.py:9]: //it executes the entire thing 8 times

Comment (xHERMIS) [reply ID to TOKEN WITH THREADING.py:9]: thats just the package xD

Comment (TheRealReed) [reply ID to TOKEN WITH THREADING.py:9]: ik it didnt use to just be that

Comment (TheRealReed) [reply ID to TOKEN WITH THREADING.py:9]: i guess it deleted some of my stuff

**Pull request closed without merging**